tools/doc/generate.js

```diff
@@ -48,11 +50,15 @@ function next(er, input) {
       break;
 
     case 'html':
-      require('./html.js').toHTML(input, inputFile, template, nodeVersion,
-        function(er, html) {
-          if (er) throw er;
-          console.log(html);
+      toHTML(input, inputFile, template, nodeVersion, (er, html) => {
+        if (er) throw er;
+        const filename = `./out/doc/api/${path.basename(inputFile, 'md')}html`;
```

Comment:
I am not a 100 % sure I like having the output paths hardcoded here:

- Generally, the doctool can be run on *any* `.md` file and is somewhat agnostic about the project it belogs to (i.e. it doesn’t even have to be Node.js docs that are being processed), and I’d prefer to keep it more or less that way
- This would make it harder to include docs from other directories if that’s desired at some point (`doc/topics` being a likely candidate?)
- Having the Makefile output include the output file makes it easier to reason about what is actually happening during building

So… what are the advantages of doing it this way?